planning/obstacle_avoidance_planner/include/obstacle_avoidance_planner/common_structs.hpp

```diff
@@ -19,6 +19,8 @@
 #include "rclcpp/rclcpp.hpp"
 #include "tier4_autoware_utils/tier4_autoware_utils.hpp"
 
+#include <boost/optional.hpp>
+
```

Comment:
Could you use std::optional instead of boost::optional?

Reply:
Thanks for the proposal @takayuki5168 . I have fixed that. Please have a look.